code/game/objects/structures/displaycase.dm

```diff
@@ -336,6 +350,10 @@
 		else
 			..()
 
+/obj/item/key/displaycase

```

Comment:
Realistically I'm only using it for easy access to the generic key sprite. If I can get a sprite I can repath it to `/obj/item/displaykey` or something.